proxy/http/Http1ClientTransaction.cc

```diff
@@ -67,6 +67,7 @@ Http1ClientTransaction::transaction_done()
   // If the parent session is not in the closed state, the destroy will not occur.
   if (parent) {
     parent->destroy();
+    parent = NULL;
```

Comment:
OK, I don't like the fragility of how this is done, but I agree this change will fix the crash.